node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-task-lib",
-  "version": "3.1.8",
+  "version": "3.1.9",
```

Comment:
Should we update package-lock also?

Reply:
I think so; `npm version patch` updates both `package.json` and `package-lock.json`.